contrib/init/systemd/docker.service

```diff
@@ -1,34 +1,47 @@
 [Unit]
 Description=Docker Application Container Engine
 Documentation=https://docs.docker.com
-After=network-online.target docker.socket firewalld.service
+After=network-online.target docker.socket firewalld.service containerd.service
```

Comment:
There was a merge conflict here, because the unit file in docker-ce-packaging did not have `docker.socket` here; that was added in this repository through https://github.com/moby/moby/pull/7257, so I opted to keep that in the list and append the `containerd.service` from the docker-ce-packaging repository